# Generated by Django 5.0.1 on 2024-02-03 18:42

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('owt_api', '0001_initial'),
    ]

    operations = [
        migrations.AlterField(
            model_name='weightrecord',
            name='body_water',
            field=models.DecimalField(decimal_places=1, max_digits=4, validators=[django.core.validators.MinValueValidator(25), django.core.validators.MaxValueValidator(70)]),
        ),
        migrations.AlterField(
            model_name='weightrecord',
            name='bone_mass',
            field=models.DecimalField(decimal_places=1, max_digits=4, validators=[django.core.validators.MinValueValidator(1), django.core.validators.MaxValueValidator(10)]),
        ),
        migrations.AlterField(
            model_name='weightrecord',
            name='fat_mass',
            field=models.DecimalField(decimal_places=1, max_digits=4, validators=[django.core.validators.MinValueValidator(5), django.core.validators.MaxValueValidator(50)]),
        ),
        migrations.AlterField(
            model_name='weightrecord',
            name='muscular_mass',
            field=models.DecimalField(decimal_places=1, max_digits=4, validators=[django.core.validators.MinValueValidator(20), django.core.validators.MaxValueValidator(100)]),
        ),
        migrations.AlterField(
            model_name='weightrecord',
            name='weight_value',
            field=models.DecimalField(decimal_places=1, max_digits=4, validators=[django.core.validators.MinValueValidator(20), django.core.validators.MaxValueValidator(250)]),
        ),
    ]
